files/firstLaunch.js

```diff
@@ -207,6 +208,156 @@ module.exports = async () => {
         })
     }
 
+    async function runSpeedtest() {
+        // write config file in librespeed-cli folder
+        const configFile = `${process.cwd()}/files/librespeed-cli/config.json`
+        const config = [
+            {
+                "id": 1,
+                "name": "o!rdrFR",
+                "server": "https://st1.issou.best/",
+                "dlURL": "garbage.php",
+                "ulURL": "empty.php",
+                "pingURL": "empty.php",
+                "getIpURL": "getIP.php"
+            }
+        ]
+        fs.writeFileSync(configFile, JSON.stringify(config))
+
+        // Run speedtest
+        console.log("Running speedtest...")
+        const speedtest = spawn(`${process.cwd()}/files/librespeed-cli/librespeed-cli` + (process.platform === "win32" ? ".exe" : ""), ["--json", "--local-json", `${process.cwd()}/files/librespeed-cli/config.json`])
+        speedtest.stdout.setEncoding("utf8")
+        speedtest.stdout.on("data", async data => {
+            const parsedData = JSON.parse(data)
+
+            console.log(`Download: ${parsedData[0].download}`)
+            console.log(`Upload: ${parsedData[0].upload}`)
+
+            // comment about upload speed
```

Comment:
we could probably change the comments about the download and upload speed, those don't really feel appropriate imo
since it's just if statements, if someone has +50mbps upload/download speed there would be the log for the speed > 10 and the log for the speed > 50 which is probably not wanted
we could instead just use something like if speed > 50 it's good enough and if speed < 50 it's a bit slow, because 10mbps is very slow for videos upload or skins/maps download